### PR TITLE
[fastlane] Added danger integration

### PR DESCRIPTION
**Pull request by KrauseFx**
oh yeah cc @orta

### CODE REVIEW

File: fastlane/lib/fastlane/actions/danger.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      end
+
+      def self.details
+        "More information: https://github.com/danger/danger"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :use_bundle_exec,
+                                       env_name: "FL_COCOAPODS_USE_BUNDLE_EXEC",
```
Comment (lacostej) [inline fastlane/lib/fastlane/actions/danger.rb]: strange env_name

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/danger.rb]: Arrg!

**Comment by KrauseFx** [comment]
Updated

**Comment by asfalcone** [comment]
LGTM 👍

**Comment by pepibumur** [comment]
👏 🎉

**Pull request merged**